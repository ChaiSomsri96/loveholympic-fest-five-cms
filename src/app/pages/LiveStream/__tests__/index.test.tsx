import * as React from 'react';
import { render } from '@testing-library/react';

import { LiveStream } from '..';

jest.mock('react-i18next', () => ({
  useTranslation: () => {
    return {
      t: str => str,
      i18n: {
        changeLanguage: () => new Promise(() => {}),
      },
    };
  },
}));

describe('<LiveStream  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<LiveStream />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
