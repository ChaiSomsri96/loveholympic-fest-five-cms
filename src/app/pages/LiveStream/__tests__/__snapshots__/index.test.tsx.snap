// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LiveStream  /> should match snapshot 1`] = `
.c0 {
  background-color: #ecf0f5;
  padding: 1rem;
  height: 100%;
}

.c1 {
  color: #2c5282;
  font-family: Roboto;
  font-size: 24px;
  font-weight: bold;
  padding-top: 1.5rem;
}

<div
  class="c0"
>
  <p
    class="c1"
  >
     
    Live Stream Management
     
  </p>
  <div
    class="row"
  >
    <div
      class="col-sm-6"
    >
      <div
        class="pb-3"
      >
        <p>
          Link 
        </p>
      </div>
      <div>
        <p>
          Item Notification
        </p>
      </div>
    </div>
    <div
      class="mt-3 col-sm-12"
    >
      <button
        class="btn btn-primary"
      >
        Save
      </button>
    </div>
  </div>
</div>
`;
